addons/mail/models/mail_thread.py

```diff
@@ -2121,7 +2121,7 @@ def message_post_with_view(self, views_or_xmlid, **kwargs):
             record.message_post_with_template(False, **kwargs)
 
     @api.multi
-    def message_post_with_template(self, template_id, **kwargs):
+    def message_post_with_template(self, template_id, auto_commit=False, **kwargs):
```

Comment:
why not fetch it from kwargs? this way no need to change the signature

Reply:
Because [kwargs is used to create a record in line 2148](https://github.com/odoo/odoo/pull/33235/files#diff-debf30f858f349749391b9d662b7eedaR2148).

Also, this is master, so changing the signature shouldn't be a problem, right? :thinking: